Dockerfile-operator

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright 2018 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM confluentinc/cp-server-connect-operator:5.3.0.0
```

Comment:
Should this be 5.3.1 (or 5.3.1.0) instead of 5.3.0.0?

Reply:
the published version in DockerHub right now is 5.3.0.0 (the extra 0 is correct, based on Operator images).

I think in a separate PR we might want to address building the various images w/ given image versions in the Makefile, or something similiar.  So it's standardized how it should work.   I say a separate PR because we have other Dockerfiles here that could use that same kind of automation/standardization work.  Thoughts?

Reply:
@rspurgeon I forgot that the demo is still on 5.3.0.0.